### PR TITLE
Update Bug report template.

### PR DESCRIPTION
**Pull request by ruebot**
**GitHub issue(s)**: none

# What does this Pull Request do?

Updating our bug report template to tease out more information that is necessary for helping troubleshoot and diagnose issues. Remove some stuff that I didn't really feel mattered. Bit of a shortcoming I noticed with #267.

# How should this be tested?

Read it, and see if you agree with it.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -20,16 +20,12 @@ A clear and concise description of what you expected to happen.
...
- - Browser: [e.g. chrome, safari]
- - Version: [e.g. 22]
-
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser: [e.g. stock browser, safari]
- - Version: [e.g. 22]
+**Environment information**
+ - aut version: [e.g. 0.16.0, HEAD]
```
Comment (ianmilligan1) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: We tend to capitalize `aut` as `AUT` throughout our docs, could we change it here for consistency?

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 2]
```diff
@@ -20,16 +20,12 @@ A clear and concise description of what you expected to happen.
...
+ - OS: [e.g. MacOS 10.13.3, Ubuntu 18.04]
+ - Java version: [e.g. Java 8]
+ - Apache Spark version: [e.g 2.1.3, 2.3.1]
+ - Apache Spark w/aut: [e.g. --jars, --packages]
```
Comment (ianmilligan1) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: maybe make even clearer what we want

`-Apache Spark command used to run AUT: [e.g ./spark-shell --driver-memory 55G --packages "io.archivesunleashed:aut:0.16.0"]`

That'd be useful to know - i.e. to get the full context of what they're doing.

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 3]
```diff
@@ -20,16 +20,13 @@ A clear and concise description of what you expected to happen.
...
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser: [e.g. stock browser, safari]
- - Version: [e.g. 22]
+**Environment information**
+ - AUT version: [e.g. 0.16.0, HEAD]
+ - OS: [e.g. MacOS 10.13.3, Ubuntu 18.04]
+ - Java version: [e.g. Java 8]
+ - Apache Spark version: [e.g 2.1.3, 2.3.1]
```
Comment (SamFritz) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: I think the changes to the template work really well @ruebot. Two tiny tweaks to offer:
e.g  --> e.g.

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 4]
```diff
@@ -20,16 +20,13 @@ A clear and concise description of what you expected to happen.
...
+ - Apache Spark w/aut: [e.g. --jars, --packages]
+ - Apache Spark command used to run AUT: [e.g ./spark-shell --driver-memory 55G --packages "io.archivesunleashed:aut:0.16.0"]
```
Comment (SamFritz) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: small spacing change: 
e.g ./ ---> e.g. /

**Pull request merged**